__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+process.env.NODE_ENV = "test";
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const app = require("../app");
+const request = require("supertest");
+const res = require("express/lib/response");
```

Comment:
Don't need this line. I suppose it's VScode trying to be helpful but not really?